spring-cloud-sleuth-zipkin/src/main/java/org/springframework/cloud/sleuth/zipkin2/ZipkinAutoConfiguration.java

```diff
@@ -68,14 +70,27 @@
 @Import({ ZipkinSenderConfigurationImportSelector.class, SamplerAutoConfiguration.class })
 public class ZipkinAutoConfiguration {
 
+	/**
+	 * Zipkin reporter bean name. Name of the bean matters for supporting multiple tracing
+	 * systems.
+	 */
+	public static final String REPORTER_BEAN_NAME = "zipkinReporter";
+
+	/**
+	 * Zipkin sender bean name. Name of the bean matters for supporting multiple tracing
+	 * systems.
+	 */
+	public static final String SENDER_BEAN_NAME = "zipkinSender";
+
 	/**
 	 * Accepts a sender so you can plug-in any standard one. Returns a Reporter so you can
 	 * also replace with a standard one.
 	 */
-	@Bean
+	@Bean(REPORTER_BEAN_NAME)
 	@ConditionalOnMissingBean
 	public Reporter<Span> reporter(ReporterMetrics reporterMetrics,
-			ZipkinProperties zipkin, Sender sender, BytesEncoder<Span> spanBytesEncoder) {
+			ZipkinProperties zipkin, @Qualifier(SENDER_BEAN_NAME) Sender sender,
```

Comment:
I think this needs a doc update associated with it since now overriding requires the correct bean name for a `Sender`

Reply:
This whole change needs a doc update because the overall configuration has been updated to support multiple tracers.
I already made the change for spring-cloud-gcp-trace (https://github.com/spring-cloud/spring-cloud-gcp/pull/1383).